src/main/java/trello/entities/List.java

```diff
@@ -124,8 +124,8 @@ public boolean areListsEquals(final ArrayList<Card> listCards) {
      * @param listToCompare is the list which is wanted to ve verified.
      * @return true if the given ArrayList of cards is sorted.
      */
-    public boolean isSortedByName(final ArrayList<Card> listToCompare) {
-        ArrayList<String> cardsTitles = getCardsTitles();
+    public boolean isSortedByName(final java.util.List<Card> listToCompare) {
+        java.util.List<String> cardsTitles = getCardsTitles();
```

Comment:
Please go up this import to the section of import.

Reply:
This is not an import. It is like that so the IDE couldn't confuse between the java.util.List and the entity List that we are using in the context.